packages/flutter_driver/test/src/real_tests/extension_test.dart

```diff
@@ -877,6 +880,108 @@ void main() {
     expect(children.single['children'], isEmpty);
   });
 
+  group('extension finders', () {
+    final Widget debugTree = Directionality(
+      textDirection: TextDirection.ltr,
+      child: Center(
+          child: Column(
+        key: const ValueKey<String>('Column'),
+        children: <Widget>[
+          const Text('Foo', key: ValueKey<String>('Text1')),
+          const Text('Bar', key: ValueKey<String>('Text2')),
+          FlatButton(
+            child: const Text('Whatever'),
+            key: const ValueKey<String>('Button'),
+            onPressed: () {},
+          ),
+        ],
+      )),
+    );
+
+    testWidgets('unknown extension finder', (WidgetTester tester) async {
+      final FlutterDriverExtension extension = FlutterDriverExtension(
+          (String arg) async => '', true,
+          finders: <FinderExtension>[]);
+
+      Future<Map<String, dynamic>> getText(SerializableFinder finder) async {
+        final Map<String, String> arguments =
+            GetText(finder, timeout: const Duration(seconds: 1)).serialize();
+        final Map<String, dynamic> response = await extension.call(arguments);
+        return response;
+      }
+
+      await tester.pumpWidget(debugTree);
+
+      final Map<String, dynamic> result = await getText(StubFinder('Text1'));
+      expect(result['isError'], true);
+      expect(result['response'] is String, true);
+      expect(result['response'] as String,
+          contains('Unsupported search specification type Stub'));
+    });
+
+    testWidgets('simple extension finder', (WidgetTester tester) async {
+      final FlutterDriverExtension extension = FlutterDriverExtension(
+          (String arg) async => '', true,
+          finders: <FinderExtension>[
+            StubFinderExtension(),
+          ]);
```

Comment:
here and elsewhere, use a trailing comma and make sure all arguments are on a new line.